src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookConsumer.java

```diff
@@ -47,6 +48,16 @@
     @Inject
     private PullRequestStore pullRequestStore;
 
+    void process(AbstractWebhookEvent e) {
```

Comment:
I'm a big fan of this change- it's a good measure to make our webhook code more generic, especially if we intend on supporting more in the future.

Reply:
Updated